onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.h

```diff
@@ -4,18 +4,21 @@
 #pragma once
 
 #include "helper.h"
+#include "qdq_support_helper.h"
 
 namespace onnxruntime {
 namespace nnapi {
 
 struct OpSupportCheckParams {
-  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw)
+  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw, QDQSupportHelper qdq_support_helper)
       : android_feature_level(android_feature_level),
```

Comment:
could this be `QDQSupportHelper&& qdq_support_helper` to avoid any unnecessary copies?